cmd/cdi-controller/controller.go

```diff
@@ -45,12 +45,12 @@ func init() {
 	// env variables
 	importerImage = os.Getenv(IMPORTER_IMAGE)
 	if importerImage == "" {
-		importerImage = IMPORTER_DEFAULT_IMAGE
+		glog.Fatalf("Environment Variable %q undefined\n")
```

Comment:
This is a backward incompatible change and thus this pr likely needs to be a version bump.

Reply:
Good point, this would push us into v2.0.0